src/components/Forecast/Widget.jsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react'
+import Form from './Form/Form'
+import WeatherDetailsToday from './WeatherDetailsToday/WeatherDetailsToday'
+import FiveDayForecast from './FiveDayForecast/FiveDayForecast'
+
+import './Widget.scss'
+
+const myAPIKey = 'dce6d076b7a2ffe64e08efe5ff779fa3'
+
+class Widget extends React.Component {
+  constructor(props) {
+    super(props)
+    this.getWeatherData = this.getWeatherData.bind(this)
+    this.goback = this.goback.bind(this)
+    this.state = {
+      cityFound: false,
+      isClicked: false,
+      city: undefined,
+      country: undefined,
+      temperature: undefined,
+      humidity: undefined,
+      description: undefined,
+      wind: undefined,
+      pressure: undefined,
+      sunrise: undefined,
+      sunrise: undefined,
+      icon: undefined,
+      in3hours: undefined,
+      in6hours: undefined,
+      in9hours: undefined,
+      in12hours: undefined,
+      day1: undefined,
+      day2: undefined,
+      day3: undefined,
+      day4: undefined,
+      day5: undefined,
+      error: undefined,
+    }
+  }
+
+  getWeatherData(event) {
+    event.preventDefault()
+
+    const city = event.target.elements.city.value
+    const country = event.target.elements.country.value
+
+    event.target.elements.city.value = ''
+    event.target.elements.country.value = ''
```

Comment:
Normally I'd suggest you let React manage these values, making the two input controlled inputs. but I know you're new to react and this is fine for this purpose 🙂 
Still, take a look at https://reactjs.org/docs/forms.html#controlled-components